src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultDynamicDocumentContainer.cs

```diff
@@ -63,5 +63,12 @@ public override IRazorDocumentPropertiesService GetDocumentPropertiesService()
             // opt out of those features.
             return null;
         }
+
+        public override IRazorDocumentOptionsProvider GetDocumentOptionsProvider()
+        {
+            // DocumentOptionsProviders are used to tell Roslyn which options to use to format edits. That information isn't needed
```

Comment:
Wasn't sure of this comment's accuracy - DefaultDynamicDocumentContainer only operates on non-open documents, is that correct?

Reply:
> Wasn't sure of this comment's accuracy - DefaultDynamicDocumentContainer only operates on non-open documents, is that correct?

Correct! We'll probably eventually want to populate options for closed documents to protect against scenarios when there's a refactoring done to an open document that also impacts a closed document. Not needed for now though